mos_tests/neutron/python_tests/base.py

```diff
@@ -141,6 +141,26 @@ def check_ping_from_vm(self, vm, vm_keypair=None, ip_to_ping=None,
         ).format(**res)
         assert 0 == res['exit_code'], error_msg
 
+    def check_no_ping_from_vm(self, vm, vm_keypair=None, ip_to_ping=None,
+                              timeout=3 * 60, vm_login='cirros',
+                              vm_password='cubswin:)'):
+        if ip_to_ping is None:
```

Comment:
Let's extract common logic (L130-L136, and L147-L153) to new method and call them from both `check_no_ping_from_vm` and `check_ping_from_vm`